README.md

```diff
@@ -55,18 +55,20 @@ have been listed below.
 * [SDL2](https://www.libsdl.org/) (needed for input, window creation and audio)
 * [OpenAL Soft](https://github.com/kcat/openal-soft) (needed for audio)
 
-At this time the project should compile for Linux (_Ubuntu 18.10_) with these dependencies 100%
+At this time the project should compile for Linux (_Ubuntu 19.10_) with these dependencies 100%
 but Windows needs some further care before it will be up and running correctly (see Windows section below).
 
-Once you've got everything you need, you'll need to compile the [extraction](https://github.com/TalonBraveInfo/OpenHoW/tree/master/src/tools/extractor) utility. In order to do this you'll need to generate your project files
-using the CMakeLists found under 'src/tools/'.
+Once compiled, you need to use te [extraction](https://github.com/TalonBraveInfo/OpenHoW/tree/master/src/tools/extractor) utility: point it to your Hogs of War installation directory and the tool will then copy across and update any of the original Hogs of War's files and assets as necessary. If you want the files to be copied over somewhere else, add `-<output-dir>` after specifying your Hogs of War directory, but the default ```bin``` directory is required to get OpenHoW up and running.
```

Comment:
* [...] to use **the** [...]
* [...] installation directory**.** The tool will [...]
* [...] Hogs of War's assets as [...] (Just removed *files*, I think just having 'assets' here is fine. Otherwise there's too many 'ands' lol.)